src/GraphQL/Types/Composite/InputObjectGraphType.cs

```diff
@@ -36,7 +36,7 @@ public class InputObjectGraphType : InputObjectGraphType<object>
     }
 
     /// <inheritdoc cref="IInputObjectGraphType"/>
-    public class InputObjectGraphType<TSourceType> : ComplexGraphType<TSourceType>, IInputObjectGraphType
+    public class InputObjectGraphType<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.NonPublicProperties | DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields)] TSourceType> : ComplexGraphType<TSourceType>, IInputObjectGraphType
```

Comment:
It's required by `value.ToObject` and inner calls from it.

Reply:
No, it isn't "required".  It's simply the warning on the inner calls within `ToObject` because .NET doesn't know that a static reference exists in the code.  We need not apply warnings to our methods where it is known that the warning does not apply.  Rather, it is correct and proper to suppress them.

Reply:
OK. I got the core idea, makes sense, but then the next question is - how to know when the warning does not **really** apply? I guess that the answer lies on the surface - one need to know well and understand the internal structure of code and its usage (usage _patterns_ I mean) by consumers. It turns out that this task is **much** more complicated than NRT annotations.

Reply:
> Rather, it is correct and proper to suppress them.

One more note. I think that suppression messages should be as much detailed as possible to keep things working while project evolves. I mean this is some sort of implicit contract between layered APIs and their consumers. This implicit contract is quite easy to break invisible for everyone. In this regard, the question arises of comprehensive testing. It would be good to somehow run all the already written tests (OK, not all but most of them) in the "AOT-ed environment". I wonder about something like _xUnit AOT runner_ 🤔 .